modules/ROOT/pages/installation/requirements.adoc

```diff
@@ -108,9 +108,19 @@ For cloud environments, and server-based, on-premise environments:
 | *Windows Server 2016, 2019, 2022*                | OracleJDK 17, ZuluJDK 17
 |===
 
+label:deprecated[Deprecated in 5.26]
```

Comment:
This is good for now, but when we have the versions of the OSs that we actually support, we can update the tables above instead.